mlflow/docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+services:
+  mlflow:
+    build: .
+    image: ghcr.io/mlflow/mlflow
+    ports:
+      - 5000:5000
+    environment:
+      MLFLOW_SERVER_DEFAULT_ARTIFACT_ROOT: /mlflow/artifacts
+    command: mlflow server --backend-store-uri postgresql://postgres:password@db:5432/mlflow --host 0.0.0.0
+  db:
+    image: postgres:latest
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: password
+      POSTGRES_DB: mlflow
```

Comment:
You should never upload credentials to the repository. I know this is just a pet project let's make a good habit regarding secrets. Common practice while working with `docker-compose` is to use environment variables with all confidential values. You can pass those values from CLI or provide `.env` file. Documentation: https://docs.docker.com/compose/how-tos/environment-variables/set-environment-variables/.